<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VNAON2C" failures="0" tests="2" name="com.qa.userTest.LoginTest" time="15.046" errors="1" timestamp="2024-01-02T11:22:23 IST" skipped="0">
  <testcase classname="com.qa.userTest.LoginTest" name="loginTest1" time="3.101"/>
  <system-out/>
  <testcase classname="com.qa.userTest.LoginTest" name="loginTest2" time="11.945">
    <error message="Page closed" type="com.microsoft.playwright.PlaywrightException">
      <![CDATA[com.microsoft.playwright.PlaywrightException: Page closed
at com.microsoft.playwright.impl.PageImpl$WaitablePageClose.get(PageImpl.java:1344)
at com.microsoft.playwright.impl.WaitableRace.get(WaitableRace.java:44)
at com.microsoft.playwright.impl.WaitableRace.get(WaitableRace.java:44)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:120)
at com.microsoft.playwright.impl.PageImpl.lambda$pause$58(PageImpl.java:935)
at com.microsoft.playwright.impl.LoggingSupport.lambda$withLogging$0(LoggingSupport.java:36)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:47)
at com.microsoft.playwright.impl.ChannelOwner.withLogging(ChannelOwner.java:89)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:35)
at com.microsoft.playwright.impl.PageImpl.pause(PageImpl.java:929)
at com.qa.userTest.LoginTest.loginTest2(LoginTest.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
]]>
    </error>
  </testcase> <!-- loginTest2 -->
  <system-out/>
</testsuite> <!-- com.qa.userTest.LoginTest -->
